pkg/server/sandbox_run.go

```diff
@@ -452,13 +453,40 @@ func (c *criService) generateSandboxContainerSpec(id string, config *runtime.Pod
 	}
 	g.SetProcessOOMScoreAdj(adj)
 
+	for passthroughAnnotationKey, passthroughAnnotationvalue := range c.getPassthroughAnnotations(config.Annotations, runtimePodAnnotations) {
```

Comment:
nit: Line too long.

Reply:
So these annotations are not passed to containers?

I think we should probably pass them to both sandbox and containers. E.g. for a runc extended feature, passing annotations to the pause container doesn't make sense.

Reply:
Done